jsk_ros_patch/laser_filters_jsk_patch/CMakeLists.txt

```diff
@@ -11,3 +11,7 @@ install(TARGETS laser_scan_filters_jsk_patch
         RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
         ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
         LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
+
+install(DIRECTORY launch/
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
```

Comment:
`DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}` ? maybe you don't need launch

Reply:
fixed install destination

Reply:
is this what you want?
```
k-okada@p51s:~/catkin_ws/ws_3rdparty$ ls install/share/laser_filters_jsk_patch/
cmake  footobject_filter_example.launch  footobject_filter_example.yaml  package.xml

```

Reply:
Yes. `footobject_filter_example.launch` is what i want.

Reply:
```
install(DIRECTORY a
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
install(DIRECTORY b/
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
install(DIRECTORY c
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/c
)
install(DIRECTORY d/
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/d
)
```
goes to 
```
ws_3rdparty/install/share/laser_filters_jsk_patch/a
└── a.launch
ws_3rdparty/install/share/laser_filters_jsk_patch/b.launch
ws_3rdparty/install/share/laser_filters_jsk_patch/c
└── c
    └── c.launch
ws_3rdparty/install/share/laser_filters_jsk_patch/d
└── d.launch
1 directory, 3 files
```
so
```
install(DIRECTORY a
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
install(DIRECTORY d/
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/d
)
```
is correct.